servers/src/test/java/tachyon/master/file/meta/InodeTreeTest.java

```diff
@@ -173,6 +173,7 @@ public void createPathTest() throws Exception {
     CommonUtils.sleepMs(10);
 
     // creating the directory path again results in no new inodes.
+    mThrown.expect(FileAlreadyExistsException.class);
```

Comment:
things below the statement triggering `mThrown.expect` will not be executed.
You probably want to refactor this test case.